packages/cdktf-cli/bin/cmds/helper/check-environment.ts

```diff
@@ -52,7 +52,7 @@ async function checkGoVersion(command: string) {
 
 async function checkNodeVersion(command: string) {
   const out = await getBinaryVersion(command, "node", "--version");
-  throwIfLowerVersion(command, "Node.js", "12.16.0", out);
+  throwIfLowerVersion(command, "Node.js", "14.0.0", out);
```

Comment:
there are also other files that need to be updated 🙈 see https://github.com/hashicorp/terraform-cdk/pull/1572#pullrequestreview-884412534

Reply:
done 👍